mlflow/server/js/src/components/CompareRunView.js

```diff
@@ -31,22 +34,34 @@ class CompareRunView extends Component {
         flex: '1',
       },
       'td-first': {
-        width: '500px',
+        width: '250px',
       },
       'th-first': {
-        width: '500px',
+        width: '250px',
       },
     };
 
+    let experiment = this.props.experiment;
+    let experimentId = experiment ? experiment.getExperimentId() : undefined;
     return (
       <div className="CompareRunView">
-        <h1>Comparing {this.props.runInfos.length} Runs</h1>
+        {experiment ?
+            <h1>
```

Comment:
Any thoughts about combining this breadcrumb and the other one and using a shared `Breadcrumb` component?